### PR TITLE
ioBroker backend

### LINKED ISSUE #707: ioBroker support.
Can you imagine the support of ioBroker additionally to openhab?

### PR DESCRIPTION
**Pull request by joltcoke**
This is WIP. Missing stuff:

- Connection retrying
- Diagram data support

Fixes #707 and https://github.com/ioBroker/AdapterRequests/issues/86

### CODE REVIEW

File: cometvisu.code-workspace
Comment (ChristianMayer) [inline cometvisu.code-workspace:1]: Is this an IDE setting?
I'm not sure whether we want IDE settings in the repository.

File: doc/manual/de/install/allgemein.rst [hunk 1]
```diff
@@ -62,6 +62,12 @@ MQTT
 Damit die CometVisu ein MQTT-Teilnehmer ist, wird ein MQTT Broker benötigt
 der über WebSocket ansprechbar ist.
 
+ioBroker
+~~~~~~~~
+
+Die Installation des "Web socket" Adapters in ioBroker ist nötig. Dieser Adapter
+wird der für den Datenaustausch verwendet.
```
Comment (ChristianMayer) [inline doc/manual/de/install/allgemein.rst:69]: Klingt komisch.

**Comment by ChristianMayer** [comment]
General code review, just by looking at the source: the files are looking fine, except two issues. See the individual comments for those.

When you think it's ready to merge just give a cheer and I'll test it (with knxd-backend; I cannot test the other backends)

File: cometvisu.code-workspace
Comment (joltcoke) [reply cometvisu.code-workspace:1]: It's a Visual Studio Code project file so you can easily open the project is VS Code. It doesn't contain any settings. Should I keep or remove the project file?

File: doc/manual/de/install/allgemein.rst (hunk 1 above)
Comment (joltcoke) [reply doc/manual/de/install/allgemein.rst:69]: Yeah, I recently updated the phrase. Still my preferred solution is a fully self contained package in the ioBroker adapter "store", I'll dig deeper how to get there after solving the diagram implementation. Will update this section then.

**Comment by joltcoke** [comment]
Thanks for your review @ChristianMayer! For now let's keep this pending as draft until the diagram implementation has been ironed out. I'm moving my own installation to ioBroker as we speak, so I should have some more confidence on the implementation quality in the next couple days.

**Comment by peuter** [comment]
@joltcoke Is this still WIP?

**Comment by peuter** [comment]
Closed due to inactivity, Feel free to re-open it if you want to continue working in this.

**Comment by joltcoke** [comment]
Superseded by #1481

**Pull request closed without merging**